CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.11.0)
+project(arrayfire-python)
+find_package(PythonExtensions REQUIRED)
+include(FetchContent)
+
+set(CMAKE_MODULE_PATH_OLD ${CMAKE_MODULE_PATH})
+set(CMAKE_MODULE_PATH "")
```

Comment:
`CMAKE_MODULE_PATH` is empty by default. Is the intent to enable usage of this `CMakeLists.txt` from another CMake project ?

Reply:
CMAKE_MODULE_PATH gets overwritten by arrayfire in the following FetchContent and then messes up the remainder of the install. We should fix upstream, but for now this will need to do.